app/uk/gov/hmrc/agentsubscriptionfrontend/views/MainTemplate.scala.html

```diff
@@ -45,21 +33,13 @@
     userIsLoggedIn: Boolean = true,
     hasTimeout: Boolean = true,
     backLinkHref: Option[String] = None
-)(mainContent: Html)(implicit request : Request[_], messages: Messages, appConfig: AppConfig)
+)(mainContent: Html)(implicit request : Request[_], msgs: Messages, appConfig: AppConfig)
 
 @beforeContent = {
     @govukPhaseBanner(standardBetaBanner(url = appConfig.betaFeedbackUrl))
```

Comment:
If the betaBanner is fed into the layout isn't this duplication? We should remove beforeContent if it is unused.
Same with the custom `components/header` file

Reply:
think it's unused so no! but i'll remove :)